### ISSUE TITLE
Do not find docker installed in the instances

### ISSUE THREAD

**Issue by meekrosoft** [kizbitz/train#71]
I have tried using train to create environments and the instances do not seem to have docker installed.  I have tried both cs and default/latest.  

```
docker run -ti --rm --env-file='train.env' -v $(pwd):/host kizbitz/train

# creating
train -vk
train -x docker

```

Can you confirm that the docker template is working?

**Comment by dougtoppin** [comment]
I just did the same (the train docker lab start with latest version of Docker, 1.11.2) and after waiting for a bit I tried it and Docker was installed and running.  If you ssh to the instance right away it may not have finished the update/install yet.
You also might need to log out and back in again in case you happen to ssh in before the update has completed and the user is added to the docker group.
Once the instance reboots it should be fully ready with Docker installed.
